src/scheduler/job_info.cpp

```diff
@@ -1778,7 +1786,7 @@ update_job_attr(int pbs_sd, resource_resv *resresv, const char *attr_name,
 
 	if (pattr != NULL && (flags & UPDATE_NOW)) {
 		int rc;
-		rc = send_attr_updates(get_svr_inst_fd(pbs_sd, resresv->job->svr_inst_id), resresv->name, pattr);
+		rc = send_attr_updates(pbs_sd, resresv->name, pattr);
```

Comment:
Don't you want to send the alter to the specific server?

Reply:
Yes, But I am not modifying pbs_asyalterjob as part of this PR as we have to handle this differently. I added a note about the same in the conversations section(last bullet point). Please check. We can take this after this PR.

Reply:
you don't have to modify pbs_asyalterjob Suresh if you just send it the right fd, right?

Reply:
without this, i don't see how this code will work for multi-server, it'll always send alterjob to the first server if we sent it the virtual fd

Reply:
We can take it up as part of #2086 once this gets merged.